hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -59,13 +65,25 @@ function ClearHyperVBindingOnAdapter($adapterName) {
     }
 }
 
+function RemoveOVSService() {
+    stop-service ovs-vswitchd
+    sc.exe delete ovs-vswitchd
+    stop-service ovsdb-server
+    sc.exe delete ovsdb-server
+}
+
 function ResetOVSService() {
```

Comment:
Found an error during running updated script:
At C:\Clean-AntreaNetwork.ps1:75 char:28 
+ function ResetOVSService() {
Missing closing '}' in statement block or type definition.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : MissingEndCurlyBrace

Reply:
updated.